dts/arm/nordic/nrf54h20_enga_cpurad.dtsi

```diff
@@ -47,5 +47,31 @@ cpuppr_vevif: &cpuppr_vevif_remote {};
 };
 
 &grtc {
-	interrupts = <109 NRF_DEFAULT_IRQ_PRIORITY>;
+	owned-channels = <7 8 9 10 11 12 13 14>;
+	child-owned-channels = <8 9 10 11 12>;
+	nonsecure-channels = <8 9 10 11 12>;
+	interrupts = <109 NRF_DEFAULT_IRQ_PRIORITY>,
+		     <108 NRF_DEFAULT_IRQ_PRIORITY>;
+};
+
+&dppic130 {
+	compatible = "nordic,nrf-dppic-global";
+	owned-channels = <0>;
+	sink-channels = <0>;
+	nonsecure-channels = <0>;
+	status = "okay";
+};
+
+&dppic132 {
+	compatible = "nordic,nrf-dppic-global";
+	owned-channels = <0>;
+	source-channels = <0>;
+	nonsecure-channels = <0>;
+	status = "okay";
+};
+
+&ipct130 {
+	owned-channels = <0>;
+	source-channel-links = <0 3 0>;
+	status = "okay";
```

Comment:
I don't like these here but I consider it as temporary workaround.
These, as of now should be enabled in case a library needs it. There is no final approach for that.
So I can change it must be enabled e.g. in case Bluetooth is required. That can be done in every BLE sample when build for the target, or... somewhere else. I will be super happy if there is a single place where I can move these to do not pollute here and copy the enable to all BLE samples.

Reply:
Maybe snippets could work? In this case, if enabling BLE for some board/SoC requires a common DT overlay, then it could be packaged into a `bluetooth` snippet containing:
```yaml
boards:
  /[^/]+/nrf54h20/cpurad/:
    append:
      EXTRA_DTC_OVERLAY_FILE: ...
```

Reply:
@57300 maybe, I have no experience with snippets. I need to read about it. If there is a way to apply it conditionally when a Kconfig is enabled that would solve the issue.

Reply:
Not really, because DT is processed before Kconfig, but the snippet itself can include a Kconfig fragment.

Reply:
Then it gets more complicated. It would be the best if that part of configuration is applied only if a library is build in. That is based on kconfig.

Question is if that is possible to add a snippet at that build stage when the library cmakelists.txt is processed.